modules/server/src/network/aggregations/AggregationAccumulator.ts

```diff
@@ -37,13 +47,19 @@ const resolveAggregations = ({ data, accumulator, requestedFields }: ResolveAggr
 		const { aggregations, hits } = data;
 
 		const isFieldAvailable = !!aggregations[requestedField];
-		const type = ALL_NETWORK_AGGREGATION_TYPES_MAP.get(requestedField);
+
+		/*
+		 * requested field will always be in ALL_NETWORK_AGGREGATION_TYPES_MAP
+		 * GQL schema validation will throw an error earlier if a requested field isn't in the schema
+		 */
+		const type = ALL_NETWORK_AGGREGATION_TYPES_MAP.get(requestedField) as SupportedAggregation;
```

Comment:
This type assertion is likely always true in your current code logic. However, if you only consider the context of this function, there is no gurantee that the `requestedField` is a known key of the `ALL_NETWORK_AGGREGATION_TYPES_MAP`. Therefore, the returned type from map.get is correct... this could be `undefined`. We should handle that undefined case, even if all it does is skip this one `requestedField` in the forEach. Likely you want to include a log statement about why the function could not complete successfully and add some details to the comment block at the top of the function explaining what is required fro this function to be used (a comment like `ALL_NETWORK_AGGREGATION_TYPES_MAP` must be initialized before this is called).